piper/ratbagd.py

```diff
@@ -550,6 +553,16 @@ def _on_properties_changed(self, proxy, changed_props, invalidated_props):
                 self._default = default
                 self.notify("is-default")
 
+    @GObject.Property
+    def capabilities(self):
+        """The capabilities of this resolution as an array. Capabilities not
+        present on the resolution are not in the list. Thus use e.g.
+
+        if resolution.CAP_DISABLE in resolution.capabilities:
+            do something
+        """
+        return self._get_dbus_property("Capabilities") or []
```

Comment:
How does this part work? What is the actual DBus call in the end? I'm super confused.